worlds/oot_soh/location_access/overworld/kokiri_forest.py

```diff
@@ -137,6 +139,31 @@ def set_region_rules(world: "SohWorld") -> None:
         (Regions.KF_STORMS_GROTTO, lambda bundle: can_open_storms_grotto(bundle))
     ])
 
+    ## KF Outside Deku Tree
+    # Locations
+    add_events(Regions.KF_OUTSIDE_DEKU_TREE, world, [
+        (EventLocations.MIDO_OUTSIDE, LocalEvents.MIDO_SWORD_AND_SHIELD, lambda bundle: has_item(Items.KOKIRI_SWORD, bundle) and
+                                                  has_item(Items.DEKU_SHIELD, bundle)),
```

Comment:
This is missing 3 events. A nut and stick event for the deku babas, and the gossip stone fairy. The latter is missing from ship (I just made a PR to ship itself to add it).